cnf-certification-test/operator/suite.go

```diff
@@ -63,6 +63,38 @@ func LoadChecks() {
 			testOperatorOlmSubscription(c, &env)
 			return nil
 		}))
+
+	checksGroup.Add(checksdb.NewCheck(identifiers.GetTestIDAndLabels(identifiers.TestOperatorHasSemanticVersioningIdentifier)).
+		WithSkipCheckFn(testhelper.GetNoOperatorsSkipFn(&env)).
+		WithCheckFn(func(c *checksdb.Check) error {
+			testOperatorSemanticVersioning(c, &env)
+			return nil
+		}))
+}
+
+// This function checks for semantic versioning of all installed operators
+func testOperatorSemanticVersioning(check *checksdb.Check, env *provider.TestEnvironment) {
+	check.LogInfo("Starting testOperatorSemanticVersioning")
+	var compliantObjects []*testhelper.ReportObject
+	var nonCompliantObjects []*testhelper.ReportObject
+
+	allOperators := env.AllOperators
+	for _, operator := range allOperators {
+		operatorVersion := operator.Version
+		check.LogInfo("Testing Operator %q for version %s", operator, operatorVersion)
+
+		if provider.IsValidSemanticVersion(operatorVersion) {
+			check.LogInfo("Operator %q has semantic version %s", operator, operatorVersion)
+			compliantObjects = append(compliantObjects, testhelper.NewOperatorReportObject(operator.Namespace, operator.Name,
+				"Operator has semantic version ", true).AddField("version", operatorVersion))
```

Comment:
`AddField("version")`: Instead of this literal "version" here, maybe you want to reuse the existing field type [Version](https://github.com/test-network-function/cnf-certification-test/blob/d63fb4f2c8da66079f80992ffaac3094312f02b6/pkg/testhelper/testhelper.go#L132).